bin/pulsar

```diff
@@ -313,6 +313,9 @@ OPTS="$OPTS -Dpulsar.functions.extra.dependencies.dir=${FUNCTIONS_EXTRA_DEPS_DIR
 OPTS="$OPTS -Dpulsar.functions.instance.classpath=${PULSAR_CLASSPATH}"
 OPTS="$OPTS -Dpulsar.functions.log.conf=${FUNCTIONS_LOG_CONF}"
 
+# Enable snappy-java to use system lib
+OPTS="$OPTS -Dorg.xerial.snappy.use.systemlib=true"
+
```

Comment:
I don't think that this should be done for the default `bin/pulsar` script. There should be another way.

Reply:
One possible solution is to append `OPTS="$OPTS -Dorg.xerial.snappy.use.systemlib=true"` to `conf/bkenv.sh` file while building the docker image. `conf/bkenv.sh` will get included for all executions of `bin/pulsar`:
https://github.com/apache/pulsar/blob/60ebab4ca6f8a45e72d1d61e29b433dde06da192/bin/pulsar#L109-L113

Reply:
A cleaner solution would be to add a feature to `bin/pulsar` so that you can pass an environment variable to specify a script that gets included sourced if a specific environment variable is set.